dockerfile/anaconda-ci/Dockerfile

```diff
@@ -90,16 +90,7 @@ RUN set -ex; \
   dnf clean all
 
 RUN pip install --no-cache-dir --upgrade pip; \
-  pip install --no-cache-dir \
-  pocketlint \
-  coverage \
-  pycodestyle \
-  dogtail \
-  rpmfluff \
-  freezegun \
-  pytest \
-  pyyaml \
-  jinja2
+  pip install --no-cache-dir -r /root/requirements.txt
```

Comment:
This means that changes in `requirements.txt` won't trigger a build of a new container if I remember it correctly. I think that there is a relevant code that needs to be updated as well.

Reply:
Fortunately, it's a bit more clever, the check is for everything in the dockerfile directory, so this will work.